packages/taquito-rpc/src/taquito-rpc.ts

```diff
@@ -1083,4 +1083,46 @@ export class RpcClient implements RpcClientInterface {
       method: 'GET',
     });
   }
+
+  /**
+   *
+   * @param contract address of the contract we want to retrieve storage information of
+   * @param options contains generic configuration for rpc calls
+   *
+   * @description Access the amount of used space used in a contract's storage
```

Comment:
looks to have an unnecessary "used" in the description
Access the used storage space of a contract